src/main/cpp/chassis/SwerveChassis.cpp

```diff
@@ -129,6 +129,9 @@ SwerveChassis::SwerveChassis(SwerveModule *frontLeft,
     m_robotConfig.numModules = moduleLocs.size();
     m_robotConfig.moduleLocations = moduleLocs;
     m_robotConfig.isHolonomic = true;
+    // TO DO Understand if we need these
+    // m_robotConfig.maxTorqueFriction = 50_Nm;
```

Comment:
I think we should initialize these to something.  Otherwise, we will get whatever is in memory at that location